tasks.py

```diff
@@ -136,7 +136,7 @@ def _detect_space(branch=None, yes=False):
 
 DEPLOY_RULES = (
     ('prod', _detect_prod),
-    ('stage', lambda _, branch: branch == 'master'),
+    ('stage', lambda _, branch: branch == 'release'),
```

Comment:
Coming back from vacation and reading this convo, I feel like I'm losing sight of what we were trying to accomplish, so I'd like to step back a bit. After reading up some more on the nitty-gritty of git-flow (I didn't realize that every change to master got tagged!), I'm beginning to think that our CD mapping should be basically what we would have had after your first line comment here:

* develop -> dev
* `startswith('release') -> staging
* master -> prod

If we wanted to "stage" a hotfix, then we could pull it into a release branch, but then it wouldn't be very "hot", would it? (The idea being that there's an urgent fix that needs to go out.) But under normal circumstances, a hotfix would only get deployed once it's merged to master (and develop). Does that seem too rash, or is it ok?